### ISSUE TITLE
enable flood pause option for synergy receive invalid option

### ISSUE THREAD

**Issue by hbxx** [HewlettPackard/POSH-HPOneView#495]
New-OVLogicalInterConnectGroup -Name name -Bays $Frames -FabricModuleType SEVC100F32 -FabricRedundancy Redundant  -FrameCount 1 -InterconnectBaySet 3  -EnableEnhancedLLDPTLV $False -EnableLLDPTagging $False -EnablePauseFloodProtection $true

in addition, when the enablepausefloodprotection option is removed, the default is set to false and it should be true

**Comment by ChrisLynchHPE** [comment]
Can you please provide more detail to the issue?  Please supply the output of `$PSLibraryVersion` and `$PSVersionTable`:

```powershell
$PSLibraryVersion | clip
$PSVersionTable | clip
```

**Comment by hbxx** [comment]
```powershell
$PSLibraryVersion
10.1.62.30                          LibraryVersion Path
----------                          -------------- ----
ApplianceVersion: 5.30.00.421400.00 5.30.2472.1534 C:\Program Files\WindowsPowerShell\Modules\HPEOneView.530\5.30.2472.1534
```

```powershell
$PSVersionTable:
Name                           Value
----                           -----
PSVersion                      5.1.17763.1007
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.1007
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
Regards
Husni

**Comment by ChrisLynchHPE** [comment]
Thanks.  I have identified the issue, and it will be fixed in the next library update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.30.2515.1313](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.30.2515.1313) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.530/5.30.2515.1313) and [Release 5.20.2515.1660](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.20.2515.1660) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2515.1660).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
